testCourse/questions/orderBlocks/question.html

```
@@ -126,5 +126,8 @@
   <pl-answer correct="true" tag="5" depends="4">By definition $a|c.$</pl-answer>
 </pl-order-blocks>
 
-
-
+<pl-order-blocks answers-name="randomize-distractors-when-ordered" grading-method="dag" feedback="first-wrong-verbose" partial-credit="lcs" source-blocks-order="ordered" randomize-distractors-when-ordered="true">
+  <pl-answer correct="true" tag="correct-block" correct="true">Correct code</pl-answer>
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ correct ] was found.

(attr-no-duplication)